django/db/backends/sqlite3/schema.py

```diff
@@ -11,26 +11,20 @@
 class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
 
     sql_delete_table = "DROP TABLE %(table)s"
-    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s)"
+    sql_create_fk = None
+    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s) DEFERRABLE INITIALLY DEFERRED"
```

Comment:
I think the text you added is appropriate. Makes me wonder if we would like to document how to migrate existing foreign keys to ``DEFERRABLE INITIALLY DEFERRED`` (which will be a bit complicated as it will require a table rebuild since SQLite doesn't support `ALTER TABLE`) or suggest [`PRAGMA defer_foreign_keys = ON`](https://sqlite.org/pragma.html#pragma_defer_foreign_keys) should be issued on connection creation.